__tests__/register-e2e-test.js

```diff
@@ -12,17 +48,17 @@ it('e2e', () => {
   )
 
   // claim a wrapper
-  experience({ owner: 'owner123' }).register(['wrapper'], async (slots, React) => {
+  await experience({ owner: 'owner123' }).register(['wrapper'], async (slots, React) => {
```

Comment:
legacy API doesn't return a promise

Reply:
you need to be using the `it('blah', cb => {})` style, the async stuff here won't actually be tested

Reply:
Thanks, will use `new Promise()` instead, looks nicer